sorc/ncep_post.fd/MISCLN.f

```diff
@@ -2392,10 +2400,11 @@ SUBROUTINE MISCLN
 !     
 !           SIGMA 0.78483 TEMPERATURE
              IF (IGET(098)>0) THEN
+               GRID1=spval
 !$omp parallel do private(i,j)
                DO J=JSTA,JEND
                  DO I=1,IM
-                   GRID1(I,J) = T78483(I,J)
+                   IF(T(I,J,LM) < spval) GRID1(I,J) = T78483(I,J)
```

Comment:
suggest change to
IF(T78483(I,J) < spval) GRID1(I,J) = T78483(I,J)

Reply:
will do

Reply:
T89671 and T78483 are computed from T and P, but will never be spval.